pkg/kapp/cmd/appchange/time_flags.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package appchange
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+type TimeFlags struct {
+	Before     string
+	After      string
+	BeforeTime time.Time
+	AfterTime  time.Time
+
+	Duration string
+}
+
+func (t *TimeFlags) Set(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&t.Before, "before", "", "", "List app-changes happened before given time stamp (format: before=2022-01-10T12:15:25Z)")
+	cmd.Flags().StringVarP(&t.After, "after", "", "", "List app-changes happened after given time stamp (format: after=2022-01-10T12:15:25Z)")
```

Comment:
I don't recall the exact use case here but the timestamp format makes me wonder if this will be "usable" 🤔

Reply:
This is what I asked also in carvel-ind once. but as we show timestamps in `kapp app-change list` user can pick the timestamps from there and apply the filter I believe.

Reply:
My thought behind going with that is that users will be copying values after listing them. I think it will be cool if we can support multiple valid formats tho. I do not think this is supported by the `time` package out of the box. 
@sethiyash today it errors out if not adhering exactly to the specific time format?

Reply:
Also, would an option to filter on _just_ date be an useful feature?
(I guess the usability of that depends on how frequently the app is deployed.)

Reply:
We can support just date also. Let's get thoughts from others also on this.